_rules/non-visual-location-reference-825ule.md

```diff
@@ -0,0 +1,506 @@
+
+## Assumptions
+
+The rule assumes that instructions take up the entirity of a single section of the content. If an instruction takes up multiple sections of the content, having a sensory characteristics in one section, and a non-sensory characteristic in another is assumed to be too disconnected to satisfy the success criterion. 
+
+The rule assumes that visual reference words are forms of information conveyed trough presentation, because of this, failing this rule fails both [Info and Relationships] and [Sensory Characteristics]. Presentation is not limited to CSS and includes images such as the image of a circle with text.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+### Understanding WCAG
+
+- [WCAG 2.1 - Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [WCAG 2.1 - Understanding Success Criterion 1.3.3: Sensory Characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html)
+
+### Related WCAG Techniques
+
+- [G96: Providing textual identification of items that otherwise rely only on sensory information to be understood](https://www.w3.org/WAI/WCAG21/Techniques/general/G96)
+- [F14: due to identifying content only by its shape or location](https://www.w3.org/WAI/WCAG21/Techniques/failures/F14)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The content in the column on the right is indicated with the word "box" (which is a shape indicator based on visual perception) but also indicated by referencing the word "howdy".
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the "howdy" box on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 2
+
+The content in the column on the right is indicated with the word "right" (which is a location indicator based on visual perception) but also indicated by referencing that the content can be found below this content in the DOM order.
+
+```html
+<head>
+<style>
+.col-container {
+  display: table;
+}
+.col {
+  display: table-cell;
+  padding: 16px;
+}
+</style>
+</head>
+<body>
+
+<div class="col-container">
+  <div class="col">
+    <p>Read the content in the right column (or below this paragraph) about the fuzzyness of cats</p>
```

Comment:
We added this example precisely because of this ambiguity. Below should pass the SC in my opinion because in this case it is (also) being used to communicate "further in the reading order".

But we might need to elaborate on that in the rule or around this specific example.

Reply:
Yes. I'm not challenging the fact that "below" should pass the SC in this case. I'm challenging the fact that I (as a potential author) would not use "below" in that specific case. 

It's not that important and likely slight improvement around this example will make that perfectly fine.